src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -99,18 +99,22 @@ SHOULD be present: For consistency between studies and institutions, we
+| ManufacturersModelName       | RECOMMENDED. Manufacturer's designation of the EEG system model (e.g., `BrainAmp DC`).                                                                                                                                                                                            |
+| SoftwareVersions             | RECOMMENDED. Manufacturer's designation of the acquisition software.                                                                                                                                                                                                              |
+| TaskDescription              | RECOMMENDED. Description of the task.                                                                                                                                                                                                                                             |
+| Instructions                 | RECOMMENDED. Text of the instructions given to participants before the scan. This is not only important for behavioral or cognitive tasks but also in resting state paradigms (e.g., to distinguish between eyes open and eyes closed).                                           |
+| CogAtlasID                   | RECOMMENDED. URL of the corresponding [Cognitive Atlas](http://www.cognitiveatlas.org/) term that describes the task (e.g., Resting State with eyes closed "[http://www.cognitiveatlas.org/task/id/trm_54e69c642d89b](http://www.cognitiveatlas.org/task/id/trm_54e69c642d89b)"). |
+| CogPOID                      | RECOMMENDED. URL of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (e.g., Rest "[http://wiki.cogpo.org/index.php?title=Rest](http://wiki.cogpo.org/index.php?title=Rest)") .                                                                       |
+| DeviceSerialNumber           | RECOMMENDED. The serial number of the equipment that produced the composite instances. A pseudonym can also be used to prevent the equipment from being identifiable, as long as each pseudonym is unique within the dataset.                                                     |
+| StimulusPresentationOS       | RECOMMENDED. Operating system used to run the stimuli presentation software.                                                                                                                                                                                                      |
+| StimulusPresentationSoftware | RECOMMENDED. Name of the software that was used to present the stimuli, ideally with its [Research Ressource Identifier](https://scicrunch.org/resources)                                                                                                                         |
```

Comment:
```suggestion
| StimulusPresentationSoftware | RECOMMENDED. Name of the software that was used to present the stimuli, ideally with its [Research Resource Identifier](https://scicrunch.org/resources)                                                                                                                         |
```

Reply:
Found a small typo ("ressource" --> "resource") that can be fixed across the tables.

Reply:
No need to apologize ! I'll spare you my elementary French, but you would find many more egregious mistakes than a small typo. Thanks for patching !

Reply:
self-deprecating commit messages / comments are OK?
https://github.com/bids-standard/bids-specification/pull/573/commits/0a00782d95665b48390e3ab9b79c648807267649